homeassistant/components/light/mochad.py

```diff
@@ -26,6 +26,9 @@
         vol.Optional(CONF_NAME): cv.string,
         vol.Required(CONF_ADDRESS): cv.x10_address,
         vol.Optional(mochad.CONF_COMM_TYPE): cv.string,
+        vol.Optional('brightness_levels',
+                     vol.In([32, 64, 256]),
+                     default=32): vol.Coerce(int),
```

Comment:
`vol.Optional('dfs', default=xy): vol.All(vol.Coerce(int), vol.In([...]))`

Reply:
oops, fixed in the new revision